book_source/06_reference/02_models/biocro.Rmd

```diff
@@ -1,31 +1,39 @@
 ## BioCro {#models-biocro}
 
-| Model Information ||
-| -- | -- |
-| Home Page | |
-| Source Code | |
-| License |  |
-| Authors | |
-| PEcAn Integration | |
+| Model Information |                                                                   |
+| --                | --                                                                |
+| Home Page         | http://biocro.r-forge.r-project.org/                              |
```

Comment:
r-forge is to an earlier version I think - a better home page might be just linking to the README in the github biocro repo.
```suggestion
| Home Page         | https://github.com/ebimodeling/biocro/blob/master/README.md |
```